docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -20,18 +21,39 @@ Sources and resources that can be loaded using this verified source are:
 | sql_database | Retrieves data from an SQL database       |
 | sql_table    | Retrieves data from an SQL database table |
 
+### Supported databases
+
+We support all [SQLAlchemy dialects](https://docs.sqlalchemy.org/en/20/dialects/), which include, but are not limited to, the following database engines:
+
+* PostgreSQL
+* MySQL
+* SQLite
+* Oracle
+* Microsoft SQL Server
+* MariaDB
+* IBM DB2 and Informix
+* Google BigQuery
+* Snowflake
+* Redshift
+* Apache Hive and Presto
+* SAP Hana
+* CockroachDB
+* Firebird
+* Teradata Vantage
+
+> Note that there many unofficial dialects, such as [DuckDB](https://duckdb.org/).
```

Comment:
Same here: it's a good place to use a note [admonition](https://docusaurus.io/docs/markdown-features/admonitions) instead of the markdown blockquote.